.ci/.e2e-tests.yaml

```diff
@@ -1,8 +1,4 @@
 ---
-FEATURES:
-  - "apache"
+SUITES:
```

Comment:
Waiting fr the CI to see this in practice. I'd like to add a secondary parallel execution per feature:
```yaml
- SUITES:
  - "helm":
    - "apm-server"
    - "filebeat"
    - "metricbeat"
  - "metricbeat":
    - "apache"
    - "metricbeat"
    - "mysql"
    - "redis"
    - "vsphere"
```

Reply:
I finally went with this approach: 
```yaml
---
SUITES:
  - suite: "helm"
    feature: "apm-server"
  - suite: "helm"
    feature: "filebeat"
  - suite: "helm"
    feature: "metricbeat"
  - suite: "metricbeat"
    feature: "apache"
  - suite: "metricbeat"
    feature: "metricbeat"
  - suite: "metricbeat"
    feature: "mysql"
  - suite: "metricbeat"
    feature: "redis"
  - suite: "metricbeat"
    feature: "vsphere"
```
which is more verbose, but only used on CI